po/pt.po

```diff
@@ -364,185 +364,183 @@ msgstr "Baixo"
 
 #: ../src/Grids/PlankGrid.vala:133
 msgid "Screen Position:"
-msgstr ""
+msgstr "Posição do ecrã:"
 
 #: ../src/Grids/PlankGrid.vala:134
 msgid "Where on the screen Plank is displayed"
-msgstr ""
+msgstr "Onde a barra é exibida no ecrã"
 
 #: ../src/Grids/PlankGrid.vala:145
 msgid "Panel"
 msgstr "Painel"
 
 #: ../src/Grids/PlankGrid.vala:148 ../src/Grids/PlankGrid.vala:165
 msgid "Centered"
-msgstr ""
+msgstr "Centrado"
 
 #: ../src/Grids/PlankGrid.vala:151
 msgid "Alignment:"
 msgstr "Alinhamento:"
 
 #: ../src/Grids/PlankGrid.vala:152
 msgid "The alignment for the dock on the monitor's edge"
-msgstr ""
+msgstr "O alinhamento para a dock no limite do monitor"
 
 #: ../src/Grids/PlankGrid.vala:168
 msgid "Item Alignment:"
-msgstr "Alinhamento dos Items"
+msgstr "Alinhamento do item:"
 
 #: ../src/Grids/PlankGrid.vala:169
 msgid "The alignment of the items in this dock if panel-mode is used"
-msgstr ""
+msgstr "O alinhamento dos itens nesta dock se for usado o modo painel"
 
 #: ../src/Grids/PlankGrid.vala:180
 msgid "Offset:"
-msgstr ""
+msgstr "Desvio:"
 
 #: ../src/Grids/PlankGrid.vala:181
 msgid "The offset (in percent) from the alignment point to shift Plank"
-msgstr ""
+msgstr "O desvio (em percentagem) do ponto de alinhamento para mudar a barra"
```

Comment:
Same here.

Reply:
Yes I know there are several times the word 'Barra' instead of 'Plank' but those those are to stay that way on purpose.
I am also the Portuguese language translator of the Plank program in the launchpad and the elementary OS in Weblate and it makes more sense for Portuguese users to be like this.

[Anotação 2020-01-19 115617](https://user-images.githubusercontent.com/16929048/72680539-d0ce6b80-3ab2-11ea-8410-4aca899f46eb.png)

Reply:
Ah I see, I'm sorry for nitpicks. :sweat_drops:

Reply:
@ryonakano don't be sorry, it was a good question to have! :) Thank you for clarifying as well Hugo!